app/furniture/marketplace/management_component.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="flex flex-wrap items-baseline">
+  <h2 class="ml-2 mt-1">
+    <%= t('marketplace.marketplace.manage_header') %>
```

Comment:
Thoughts on using a relative path? That way if we delete the component we know what locales keys we can get rid of too?

Reply:
I like that idea a lot. I'll come up with something in a follow up.